sequence_processing_pipeline/ConvertJob.py

```diff
@@ -146,6 +151,9 @@ def run(self, callback=None):
             job_info = self.submit_job(self.job_script_path,
                                        exec_from=self.log_path,
                                        callback=callback)
+
+            self._get_sample_sheet_info()
```

Comment:
Just to confirm, can this be called before the previous line or not (before submitting jobs)? Also, is the plan to unify this functionality in a single place? Maybe `_get_sample_sheet_info()`?

Reply:
This function should only be called after the bcl-convert job returns completed. The output from this method is specific to supporting ConvertJob so there currently isn't a value in pushing it down into the base class. It's name does begin with '_' and so by convention it's an internal method and nobody should be calling it except ConvertJob.run().